control-plane/subcommand/gateway-resources/command.go

```diff
@@ -16,23 +16,28 @@ import (
 
 	"github.com/cenkalti/backoff"
 	"github.com/mitchellh/cli"
-	"gopkg.in/yaml.v2"
+	"gopkg.in/yaml.v3"
 	corev1 "k8s.io/api/core/v1"
-	v1 "k8s.io/api/core/v1"
```

Comment:
we were double importing the `core/v1` dep as `corev1` and `v1` so I changed this to only import `corev1`